Example/AblyChatExample/ContentView.swift

```diff
@@ -1,24 +1,169 @@
 import AblyChat
 import SwiftUI
 
-struct ContentView: View {
-    /// Just used to check that we can successfully import and use the AblyChat library. TODO remove this once we start building the library
-    @State private var ablyChatClient = DefaultChatClient(
-        realtime: MockRealtime.create(),
-        clientOptions: ClientOptions()
-    )
+extension MockChatClient {
+    static let shared = MockChatClient(realtime: MockRealtime.create(), clientOptions: .init())
```

Comment:
Why a singleton? Can't an instance be managed by the view?

Reply:
https://github.com/ably-labs/ably-chat-swift/pull/34/commits/b571d2720eec7bb2390385f3ade0e2f783ef0b36